v2/tools/generator/internal/astmodel/code_generation_context.go

```diff
@@ -83,34 +85,62 @@ func (codeGenContext *CodeGenerationContext) MustGetImportedPackageName(referenc
 }
 
 // GetGeneratedPackage gets a reference to the PackageDefinition referred to by the provided reference
-func (codeGenContext *CodeGenerationContext) GetGeneratedPackage(reference PackageReference) (*PackageDefinition, error) {
+func (ctx *CodeGenerationContext) GetGeneratedPackage(reference PackageReference) (*PackageDefinition, error) {
 	// Make sure that we're actually importing that package -- don't want to allow references to things we aren't importing
-	_, err := codeGenContext.GetImportedPackageName(reference)
-	if !reference.Equals(codeGenContext.currentPackage) && err != nil {
+	_, err := ctx.GetImportedPackageName(reference)
+	if !reference.Equals(ctx.currentPackage) && err != nil {
 		return nil, err
 	}
 
-	packageDef, ok := codeGenContext.generatedPackages[reference]
+	packageDef, ok := ctx.generatedPackages[reference]
 	if !ok {
 		return nil, errors.Errorf("%s not imported", reference)
 	}
 	return packageDef, nil
 }
 
-// GetImportedDefinition looks up a particular type definition in a package available in this context
-func (codeGenContext *CodeGenerationContext) GetImportedDefinition(typeName TypeName) (TypeDefinition, error) {
-	pkg, err := codeGenContext.GetGeneratedPackage(typeName.PackageReference)
+// GetOrErr looks up a particular type definition in a package available in this context
+func (ctx *CodeGenerationContext) GetOrErr(name TypeName) (TypeDefinition, error) {
+	pkg, err := ctx.GetGeneratedPackage(name.PackageReference)
 
 	if err != nil {
 		return TypeDefinition{}, err
 	}
 
-	return pkg.GetDefinition(typeName)
+	return pkg.GetDefinition(name)
+}
+
+// Get looks up a particular type definition in a package available in this context
+func (ctx *CodeGenerationContext) Get(name TypeName) TypeDefinition {
```

Comment:
This one would then be renamed `MustGetImportedDefinition()` or `MustLookupDefinition()` - and probably should panic if there is an error, not ignore it.